src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -244,7 +245,14 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
 
     communicator?.on(Methods.sendTransactions, (msg) => {
       // @ts-expect-error explore ways to fix this
-      openConfirmationModal(msg.data.params.txs as Transaction[], msg.data.params.params, msg.data.id)
+      const transctions = (msg.data.params.txs as Transaction[]).map(({ to, ...rest }) => ({
+        to: checksumAddress(to),
```

Comment:
@mikheevm, we should probably include this on the SDK side as well. What do you think?

Reply:
`checksumAddress` returns an empty string and logs an error to the console.

What happens with the transaction then? It will fail to even send?

Reply:
AFAIU, throwing in that event handler won't let the Safe App handle this error.

Reply:
@mikheevm, do we have a way of notifying the Safe App?

Reply:
There's a method, `isValidTransaction` validates the transaction before showing the modal. checksum check is not included there and I wouldn't add it because it may break a lot of apps

Reply:
@katspaugh, before confirming the transaction the validity is checked. If the checksumming function returned an empty string, [it wouldn't be possible to confirm it](https://github.com/gnosis/safe-react/blob/76d37e6d17f4ee73c2e08c4b90ad6c42b3941fb6/src/routes/safe/components/Apps/components/ConfirmTxModal/index.tsx#L36).

Reply:
Sounds good!